GitUI/UserControls/RevisionGridClasses/DvcsGraph.cs

```diff
@@ -280,6 +288,7 @@ public string[] SelectedIds
             }
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Reliability", "CA2002:DoNotLockOnObjectsWithWeakIdentity")]
```

Comment:
:bulb: I'm not convinced this is correct. I would prefer to see it moved to its own pull request and evaluated separately.